css/Comments.css

```diff
@@ -0,0 +1,3 @@
+.bulkmanagerheading .title {
```

Comment:
Right...

Well I want to PR the bulk actions module to allow turning that description off. That's a much better solution than this CSS.